javascript-arrays/main.js

```diff
@@ -0,0 +1,19 @@
+const colors = ['red', 'white', 'blue'];
+
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+console.log(`America is ${colors[0]}, ${colors[1]}, ${colors[2]}`);
+
+colors.pop();
+colors.push('green');
```

Comment:
You could also just assign the string directly like this: `colors[2] = 'green';`

Reply:
Ah! I was gonna ask you if there was a way to refactor it when we were going to review the exercise.